hat/assets/js/apps/Iaso/domains/dataSources/hooks/useGetDataSources.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable camelcase */
+import { UseQueryResult } from 'react-query';
+import { getRequest } from '../../../libs/Api';
+import { useSnackQuery } from '../../../libs/apiHooks';
+import { DataSource } from '../types/dataSources';
+
+const getDataSource = async (
+    sourceId: string | undefined,
+): Promise<DataSource> => {
+    return getRequest(`/api/datasources/${sourceId}/`);
+};
+
+export const useGetDataSource = (
+    sourceId: string | undefined,
+): UseQueryResult<DataSource, Error> => {
+    const queryKey: any[] = ['dataSource', sourceId];
+    return useSnackQuery({
+        queryKey,
+        queryFn: () => getDataSource(sourceId),
+        snackErrorMsg: undefined,
+        options: {
+            retry: false,
+            staleTime: Infinity,
```

Comment:
This means it will never refetch data until we reload the page or invalidate it. We are using a real value on other calls, maybe use the same